### ISSUE TITLE
Please use GtkFileChooserNative/GtkFileChooserButton instead of GtkFileChooserDialog/GtkFileChooserWidget

### ISSUE THREAD

**Issue by metal450** [buchen/portfolio#1693]
I'm using this solution to replace GTK file chooser dialogs with KDE ones, to present a unified behavior on KDE: https://superuser.com/a/1373504

However, this does not work for apps using the old GtkFileChooserDialog or GtkFileChooserWidget APIs (these expose the underlying GtkWidgets to the application which would not be available when rendering the file chooser using anything other than GTK). Only GtkFileChooserNative and GtkFileChooserButton will work.  Since PortfolioPerformance is the only app on my system that's still showing GTK apps...I can only presume that this is the case here, & would like to request using GtkFileChooserNative instead :)

**Comment by buchen** [comment]
I am not developing against any Linux/GTK code myself, but I am using SWT which gives me an abstraction to Windows, Linux, and macOS.

Are you building PP locally? Just yesterday, I have updated SWT to the latest version (updating from a 2 year old version). Maybe this also updates the API usage on GTK. If you do not build locally, I could share one tonight or tomorrow for you to test.

**Comment by metal450** [comment]
Nope, I'm not building locally - but sure, I could test it out :)

**Comment by mierin12** [comment]
Fixed in [release 0.48.0](https://github.com/portfolio-performance/portfolio/releases/tag/0.48.0)
